src/lib/tabs/tab.ts

```diff
@@ -65,6 +65,12 @@ export class MatTab extends _MatTabMixinBase implements OnInit, CanDisable, OnCh
    * Will be cleared if `aria-label` is set at the same time.
    */
   @Input('aria-labelledby') ariaLabelledby: string;
+  
+  /**
+   * Unique identifier for each tab.
+   * Used to differenciated which tab is being changed or switched to.
```

Comment:
```suggestion
   * Used to differentiate which tab is being changed or switched to.
```

Nitpick: `differenciated` should be `differentiate`

(Or maybe `for differentiating` instead? I'm not sure with this.)

```suggestion
   * Used for differentiating which tab is being changed or switched to.

Reply:
I was meant to type differentiate. So, I'll stick with differentiate.